### PR TITLE
[IT-2692] Create org-sagebase-scce-dev account

### PR DESCRIPTION
**Pull request by jesusaurus**
Create an account for SCCE members to explore AWS services.

### CODE REVIEW

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -627,6 +627,19 @@ Organization:
...
+  ScceDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-scce-dev
+      RootEmail: aws-scce-dev@sagebase.org
+      Alias: org-sagebase-scce-dev
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        AccountOwner: thomas.yu@sagebase.org
+        CostCenter: NO PROGRAM / 000000
```
Comment (jesusaurus) [inline org-formation/organization.yaml:639]: @thomasyu888 should this SCCE dev account have a specific `CostCenter`?

File: org-formation/organization.yaml [hunk 2]
```diff
@@ -627,6 +627,19 @@ Organization:
         budget-alarm-threshold: 10000
         budget-alarm-threshold-email-recipient: aws-dpe-prod@sagebase.org
 
+  ScceDevAccount:
```
Comment (zaro0508) [inline org-formation/organization.yaml:630]: please add this to `ScienceDevOU` above

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/organization.yaml (hunk 2 above)
Comment (jesusaurus) [reply org-formation/organization.yaml:630]: good catch, thanks

**Review by zaro0508 — APPROVED** [review]
there are several manual steps required to fully create an account. please follow directions on the wiki

File: org-formation/organization.yaml (hunk 1 above)
Comment (thomasyu888) [reply org-formation/organization.yaml:640]: No - this is just for SCCE team to explore resources.

**Review by thomasyu888 — APPROVED** [review]
💯 Thanks!

**Pull request merged**